### ISSUE TITLE
Minimum IAM access required for the Amplify Role to Host WebApp without backend

### ISSUE THREAD

**Issue by akanungo22** [aws-amplify/amplify-console#2153]
**Please describe which feature you have a question about?**

**Provide additional details**

**What AWS Services are you utilizing?**

**Provide additional details e.g. code snippets**

**Comment by Athena96** [comment]
Hi @akanungo22 if you do not have a backend and your app is not an SSR Next.js app, then you do not need a service role. Are you facing permissions issues while trying to build?

**Comment by akanungo22** [comment]
Yup, the build did not work, after I created service role and assigned to Amplify app, then it started building.
We have react app hosted.

**Comment by Athena96** [comment]
Can you please share your appId and we can investigate

**Comment by hughesjj** [comment]
If we are an SSR Next.js app, what would the minimum permissions on the backend be?
